problems/arrayMethods.js

```diff
@@ -82,7 +125,15 @@ const isAllOdd = () => {};
  * @returns {number} Number of target occurrences
  */
 
-const targetCount = () => {};
+const targetCount = (nums, target) => {
+  let counter = []
```

Comment:
Same as before. If we're keeping a count why store things in an array. We've got numbers :-).